tests/template_tests/syntax_tests/test_cache.py

```diff
@@ -122,6 +122,14 @@ def test_cache18(self):
         output = self.engine.render_to_string('cache18')
         self.assertEqual(output, 'cache18')
 
+    @setup({'cache19': '{% load cache %}{% cache None test %}cache19{% endcache %}'})
+    def test_cache19(self):
+        """
+        Allow cache timeout to be None (#27882).
```

Comment:
Yep, I can check that the content makes it into the cache. FYI it looks like most of the other cache tag tests don't test this either; maybe that can be fixed in another PR.